package.json

```diff
@@ -92,6 +92,7 @@
     "@emotion/styled": "10.0.10",
     "date-fns": "1.30.1",
     "keycode": "2.2.0",
+    "localforage": "1.9.0",
```

Comment:
See other comments about using this lib.

I would suggest creating very simple getter/setter functions based on [this minimal example](https://gist.github.com/JamesMessinger/a0d6389a5d0e3a24814b) - but using Promises so we can call them with `await` instead of passing callbacks around, so the usage would be the same as currently: `await db.getLocal(...)`. Also, I don't think we need the index.

For example, something like this (note that I haven't tested this, nor changed it to const, single quotes, etc):
```javascript
getLocal (key) {
  return new Promise((resolve, reject) => {
    // Start a new transaction
    var db = open.result;
    var tx = db.transaction("MyObjectStore", "readwrite");
    var store = tx.objectStore("MyObjectStore");

    // Query the data
    const query = store.get(key);
    query.onsuccess = function() {
      resolve(query.result);
    };

    // Close the db when the transaction is done
    tx.oncomplete = function() {
      db.close();
    };
  })
}
```

Then something similar for the setter, and this on the initialization:
```javascript
// Open (or create) the database
var open = indexedDB.open("MyDatabase", 1);

// Create the schema
open.onupgradeneeded = function() {
    var db = open.result;
    var store = db.createObjectStore("MyObjectStore", {keyPath: "id"});
};
```